samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -352,6 +352,11 @@ static void button_changed(uint32_t button_state, uint32_t has_changed)
 	bond_remove_btn_handle(button_state, has_changed);
 }
 
+static void fp_account_key_list_changed(struct bt_conn *conn)
+{
+	LOG_INF("Fast Pair Account Key list has changed");
```

Comment:
You could also denote if this was an Account Key write from a given connection or factory reset.